src/Microsoft.Health.Dicom.Api/Microsoft.Health.Dicom.Api.csproj

```diff
@@ -14,6 +14,7 @@
     <PackageReference Include="fo-dicom" Version="4.0.7" />
     <PackageReference Include="fo-dicom.Json" Version="4.0.7" NoWarn="NU1701" />
     <PackageReference Include="MediatR" Version="9.0.0" />
+    <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.17.0" />
```

Comment:
I was wrong! Looks like we need some additional package specifications for Component Governance for Microsoft.AspNetCore.Http and System.Text.Encodings.Web:

```xml
<PackageReference Include="Microsoft.AspNetCore.Http" Version="2.1.22" PrivateAssets="All" />
<PackageReference Include="System.Text.Encodings.Web" Version="4.5.1" PrivateAssets="All" />
```

You can double-check this by checking the build's CG alerts inside of the build logs.

Reply:
Added them in